src/main/java/frc/robot/subsystems/BuddyBalanceSubsystem.java

```diff
@@ -75,36 +87,92 @@ public BuddyBalanceSubsystem() {
+    refPointServoTuner = new TunableNumber("Ref Point Servo", kServoDeployedPos, (a) -> {tunedServoDeployedPos = Units.degreesToRadians(a);});
+  }
 
-    positionEntry = buddyBalanceTab.add("Buddy Balance Position", 0).getEntry();
+  public double getRightAngle() {
+    return Math.round(AbsoluteEncoder.getPositionRadians(rightEncoder)*100)/100.0;
+  }
+
+  public double getLeftAngle() {
+    return Math.round(AbsoluteEncoder.getPositionRadians(leftEncoder)*100)/100.0;
   }
 
   public boolean getIsDeployed() {
     return isDeployed;
   }
 
   public void deployBuddyBalance() {
-    activateDeploy.set(kServoDeployedPos);
+    activateDeploy.set(kServoDeployedPos); // Change constants to the tuned values while testing (kServoDeployedPos becomes tunedServoDeployedPos)
     isDeployed = true;
   }
 
   public void retrieveBuddy() { // Used to pick up the buddy robot while the lift is already underneath it
-    rightPIDController.setReference(kBalancedPosition, CANSparkMax.ControlType.kPosition);
-    leftPIDController.setReference(kBalancedPosition, CANSparkMax.ControlType.kPosition);
+    rightPIDController.setSetpoint(kRetrievedAngle); // Change constants to the tuned values while testing (KRetrievedAngle becomes tunedRetrievedAngle)
+    leftPIDController.setSetpoint(kRetrievedAngle);
+    //buddyBalancePosEntry.setString("Raised");
   }
 
   public void releaseBuddy() { // Used to set down the robot
-    rightPIDController.setReference(kDeployedPosition, CANSparkMax.ControlType.kPosition);
-    leftPIDController.setReference(kDeployedPosition, CANSparkMax.ControlType.kPosition);
+    rightPIDController.setSetpoint(kDeployedAngle); // Change constants to the tuned values while testing (kDeployedAngle becomes tunedDeployedAngle)
+    leftPIDController.setSetpoint(kDeployedAngle);
+    //buddyBalancePosEntry.setString("Lowered");
+  }
+
+  public void updateMotors() {
+    encoderCalculatedRightAngle = rightPIDController.calculate(getRightAngle());
+    encoderCalculatedLeftAngle = leftPIDController.calculate(getLeftAngle());
+    rightMotor.set(MathUtil.clamp(encoderCalculatedRightAngle, -1, 1));
+    leftMotor.set(MathUtil.clamp(encoderCalculatedLeftAngle, -1, 1));
+  }
+
+  // These methods are used for JUnit testing only
+  public void close() throws Exception {
+    // This method will close all device handles used by this object and release any other dynamic memory.
+    // Mostly for JUnit tests
+    rightMotor.close();
+    leftMotor.close();
+    activateDeploy.close();
+    rightEncoder.close();
+    leftEncoder.close();
+  }
+
+  public double getActualServoPosition() {
+    return servoActualPosition;
+  }
+
+  public static double getDesiredServoPosition() {
+    return kServoDeployedPos;
+  }
+
+  public double getActualEncoderRightAngle() {
+    return encoderActualRightAngle;
+  }
+
+  public double getActualEncoderLeftAngle() {
+    return encoderActualLeftAngle;
+  }
```

Comment:
These should return .get calls, not variables

Reply:
We initially created the variables because we believed the methods would be static but now they're unnecessary